include/service_control_client.h

```diff
@@ -281,6 +281,17 @@ class ServiceControlClient {
       DoneCallback on_check_done, TransportCheckFunc check_transport) = 0;
 
   // An async quota call.
+  // on_quota_done is called with the quota request status after cached
```

Comment:
"When cache hit occurs, `on_quota_done` is called with cached `quota_response`."

Reply:
Thanks for pointing out! I updated the comments according to your suggestion. More precisely, `on_quota_done` is only called with a status. When quota cache is enabled, the status is always OK. `quota_response` is filled in with the cached response.